CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+Please be sure to read the contribution guidelines before making or requesting a change.
+
+## Code of Conduct
+
+This project adheres to the [OpenJS Foundation Code of Conduct](https://eslint.org/conduct). We kindly request that you read over our code of conduct before contributing.
+
+## Commands
+
+### Building
+
+[Rollup](https://rollupjs.org) and [TypeScript](https://www.typescriptlang.org) are used to turn source files in `packages/*/src/` into outputs in `packages/*/lib/`.
+
+```shell
+npm run build
+```
+
+### Linting
+
+ESLint is linted using ESLint.
+[Building](#building) the project must be done before it can lint itself.
```

Comment:
Did you find this to be true? Linting generally happens on source files, so not sure why building would be required first.

Reply:
Yes, it has a dependency on its own files right now:

```plaintext
$ npm run lint

> eslint-rewrite@1.0.0 lint
> eslint .


Oops! Something went wrong! :(

ESLint: 9.9.1

Error: Cannot find module '/Users/josh/repos/rewrite/node_modules/@eslint/config-array/dist/cjs/index.cjs'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1256:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1244:15)
    at resolveExports (node:internal/modules/cjs/loader:628:14)
    at Module._findPath (node:internal/modules/cjs/loader:718:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1205:27)
    at Module._load (node:internal/modules/cjs/loader:1045:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:215:24)
    at Module.require (node:internal/modules/cjs/loader:1304:12)
    at require (node:internal/modules/helpers:123:16)
```

Running a build fixes that:

```plaintext
$ npm run build

# ... build output

 $ npm run lint

> eslint-rewrite@1.0.0 lint
> eslint .

$ |
```